br-ext/package/optee_test_ext/optee_test_ext.mk

```diff
@@ -6,10 +6,19 @@ OPTEE_TEST_EXT_INSTALL_STAGING = YES
 OPTEE_TEST_EXT_DEPENDENCIES = optee_client_ext openssl host-python3-pycryptodomex
 OPTEE_TEST_EXT_SDK = $(BR2_PACKAGE_OPTEE_TEST_EXT_SDK)
 OPTEE_TEST_EXT_CONF_OPTS = -DOPTEE_TEST_SDK=$(OPTEE_TEST_EXT_SDK)
-# os_test has dependencies, this enforces a valid build order
```

Comment:
Why remove the comment? I'm afraid that without it, someone might be tempted to replace the list with a simple wildcard `*`. In any case this does not belong to this PR.